docs/_pages/releases.md

```diff
@@ -23,6 +23,7 @@ sidebar:
 * `BeEquivalentTo` can again compare a non-generic `IDictionary` with a generic one - [#2358](https://github.com/fluentassertions/fluentassertions/pull/2358)
 * Fixed that the `FormattingOptions` were not respected in inner `AssertionScope` - [#2329](https://github.com/fluentassertions/fluentassertions/pull/2329)
 * Capitalize `true` and `false` in failure messages and make them formattable to a custom `BooleanFormatter` - [#2390](https://github.com/fluentassertions/fluentassertions/pull/2390), [#2393](https://github.com/fluentassertions/fluentassertions/pull/2393)
+* Fixed that an assertion on an exception type doesn't continue asserting the concrete type when wrapped into an assertion scope and the exception is `null` - [#2395](https://github.com/fluentassertions/fluentassertions/pull/2390), [#2395](https://github.com/fluentassertions/fluentassertions/pull/2395)
```

Comment:
These release notes are meant to be at functional level, so I try formulate the sentences such that they are understandable to an FA user that has no knowledge of the FA internals.
The exception to this guideline is the "for extensions" section where it again makes sense to mention the the deep level APIs.

```suggestion
* Improved the failure message for `ThrowExactly[Async]` when wrapped in an `AssertionScope` and no exception is thrown - [#2398](https://github.com/fluentassertions/fluentassertions/pull/2398)

Reply:
You are completely right :)

And.. if someone want details about what exactly was the problem... there is a link beside :D